config/routes.rb

```diff
@@ -42,8 +42,11 @@
         get(:healthcheck)
       end
     end
-    resource :pension_claims, only: [:create]
-    resource :burial_claims, only: [:create]
+
+    if Settings.pension_burial.upload.enabled?
```

Comment:
enabled shouldn't have a `?`

Reply:
Yeah, just realized that. Pushed it up before my specs finished... Thanks for catching it.